.github/workflows/unit-test-on-pull-request.yml

```
@@ -67,6 +67,28 @@ jobs:
       - name: Tests
         run: make test TARGET_ARCH=${{ matrix.target_arch }}
 
+  check-ebpf-blobs:
+    name: Check for differences in the eBPF binary blobs
+    runs-on: ubuntu-24.04
+    container: otel/opentelemetry-ebpf-profiler-dev:latest
+    steps:
+      - name: Clone code
+        uses: actions/checkout@v4
+      - name: Hash eBPF blobs
+        run: |
+          sha256sum support/ebpf/tracer.ebpf.release.a* > ebpf-blobs.hash
+      - name: Rebuild eBPF blobs
+        run: |
+          make amd64 -C support/ebpf
```

```suggestion
          rm support/ebpf/tracer.ebpf.release.*
          make amd64 -C support/ebpf
```
Otherwise, you can trick `make` with a future timestamp for `support/ebpf/tracer.ebpf.release.*` files.

Example, where only the .o files are created, but not the binary blobs:
```
touch --date "2026-01-14" support/ebpf/tracer.ebpf.release.*
make clean -C support/ebpf
make arm64 -C support/ebpf
```